tests/DoctrineORMModuleTest/Yuml/MetadataGrapherTest.php

```diff
@@ -400,4 +400,74 @@ function ($assoc) {
             $this->grapher->generateFromMetadata(array($class1, $class2))
         );
     }
+
+    /**
+     * @covers \DoctrineORMModule\Yuml\MetadataGrapher
+     */
+    public function testBugTestCaseFix()
```

Comment:
This test verifies multiple cases - please consider chunking it into smaller tests explaining each scenario carefully in the method name (such as `testAssociationOneToManyBlaBlaDoesDoBlaBla`)